### ISSUE TITLE
Foundation: is it possible to start a foundation for this codebase?

### ISSUE THREAD

**Issue by EricBright** [tenacityteam/tenacity-legacy#28]
**The Problem**
I am still not able to understand how it was possible for an open source project on GitHub to be sold, money be pocketed, and the rest. How could that Happen? The code did not belong to the maintainers; not all of it. How come a set of maintainers could sell a codebase written by volunteers? It only makes sense when/if the codebase actually **belonged** to them. If a person or a group of people are the “owners” of a project, then they can sell it to someone else and keep the money for themselves.

Technical legal issues, indeed! For this not to happen again, there needs to be an independent and unsellable entity that would be the steward of the code. In the case of LibreOffice, they made The Document Foundation. It plays the role of an organization that cannot be bought or sold according to the laws and regulations in Germany.

**Suggested solution**
Would it be possible, within our means in any significant way, to register a foundation in a reputable country to protect this project against a similar fate?

I suggest that a few who know these matters of legal complexities to look into the possibility of establishing a foundation for this on-going operation.

Your suggestions would be of great importance!

**Comment by panos** [comment]
... We should really get the name going first before anyone interested in this decides to take up on a role involving non-profit management. 

It's worth checking out the [Software Freedom Conservancy](https://sfconservancy.org/).

**Comment by EricBright** [comment]
Thank you for the link! Very eye-opening articles (e.g. [It Matters Who Owns Your Copylefted Copyrights](https://sfconservancy.org/blog/2021/jun/30/who-should-own-foss-copyrights/)).

**Comment by Be-ing** [comment]
Conservancy does not accept new projects. Only established projects with an existing community can join Conservancy.

Establishing a foundation is really premature. There are barely any code commits and not even a name.

**Comment by panos** [comment]
Yeah, that's why I pretty much brought it up for the long-term future.

**Comment by n0toose** [comment]
> Conservancy does not accept new projects. Only established projects with an existing community can join Conservancy.

Yeah, I just left this for later down the line. You never know. It could also be considerably easier to do this than found a non-profit organization, which is considerably hard as far as the paperwork is concerned and in jurisdictions like Germany.
 
> Establishing a foundation is really premature. There are barely any code commits and not even a name.

This is absolutely correct, in my opinion.

If this takes off, we may need to financially support some contributors and finance infrastructure (e.g. MediaWiki). Still, priorities.

**Comment by adrianheine** [comment]
> Establishing a foundation is really premature. There are barely any code commits and not even a name.

I somewhat disagree. It is probably premature, true, but in the beginning community and governance is this fork's main feature, not code changes.

**Comment by n0toose** [comment]
> beginning community and governance is this fork's main feature, not code changes.

Agreed. We can't afford to repeat the same mistakes as Audacity's, but there's multiple people working on this and I think that we get along very well together. However, Cookie is seen as the "face" of the fork right now and we are getting a distracting amount of negative attention, but we are finding ways to work around it and get things done. Just a bit slower, for the start.

**Comment by sorvani** [comment]
> I am still not able to understand how it was possible for an open source project on GitHub to be sold, money be pocketed, and the rest. How could that Happen?

Because a trademark was granted to a single individual in 2004. The trademark is what was sold.
https://en.wikipedia.org/wiki/Audacity_(audio_editor)#Audacity_trademarks


>  but in the beginning community and governance is this fork's main feature, not code changes.

I 100% agree on this. I will not change from my current 2.X line of Audacity to any fork that has no governance.
